upgrade/sql/1.7.2.0.sql

```diff
@@ -3,6 +3,8 @@ SET NAMES 'utf8';
 
 ALTER TABLE `PREFIX_store` MODIFY `hours` text;
 
+INSERT INTO `PREFIX_configuration` (`name`, `value`, `date_add`, `date_upd`) VALUES ('PS_PRODUCT_SHORT_DESC_LIMIT', '800', NOW(), NOW());
```

Comment:
You'll need to rebase in order to retrieve #1155, then you could write

```suggestion
/* PHP:add_configuration_if_not_exists('PS_PRODUCT_SHORT_DESC_LIMIT', '800'); */;
```